index.bs

```diff
@@ -281,7 +281,7 @@ When the <dfn method for="XR">supportsSession(|mode|)</dfn> method is invoked, i
   1. Run the following steps [=in parallel=]:
     1. [=ensures an XR device is selected|Ensure an XR device is selected=].
     1. If the [=XR/XR device=] is null, [=reject=] |promise| with a "{{NotSupportedError}}" {{DOMException}} and abort these steps.
-    1. If the [=XR/XR device=]'s [=list of supported modes=] does not [=list/contain=] |mode|, [=reject=] |promise| with a "{{NotSupportedError}}" {{DOMException}} and abort these steps.
+    1. If the [=XR/XR device=]'s [=list of supported immersive modes=] does not [=list/contain=] |mode|, [=reject=] |promise| with a "{{NotSupportedError}}" {{DOMException}} and abort these steps.
```

Comment:
Maybe I'm missing something here... but won't this cause `supportsSession('inline')` to be rejected?

Reply:
Fixed